Arknights Adapter Error

Here is another error, I guess this program broke while you were inactive...
This time it's Arknights Adapter Error

```
Loading default config first
Found custom config file, loading it...
Starting Arknights with name Texas
Registered on boot
Getting Arknights character data
Loading chwd data...
Loading character data...
Loading skins...
Skins loaded
Character data loaded
Exception in thread "Thread-1" com.google.gson.JsonSyntaxException: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 132200 column 23 path $.
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:226)
	at com.google.gson.internal.bind.TypeAdapterRuntimeTypeWrapper.read(TypeAdapterRuntimeTypeWrapper.java:41)
	at com.google.gson.internal.bind.MapTypeAdapterFactory$Adapter.read(MapTypeAdapterFactory.java:187)
	at com.google.gson.internal.bind.MapTypeAdapterFactory$Adapter.read(MapTypeAdapterFactory.java:145)
	at com.google.gson.Gson.fromJson(Gson.java:927)
	at com.google.gson.Gson.fromJson(Gson.java:892)
	at com.google.gson.Gson.fromJson(Gson.java:841)
	at com.google.gson.Gson.fromJson(Gson.java:813)
	at com.kitsunecode.mms.core.adapters.impl.adapterentities.arknights.charword.CharwordMap.fromJson(CharwordMap.java:13)
	at com.kitsunecode.mms.core.adapters.impl.Arknights.lambda$loadFromCustomSource$0(Arknights.java:52)
	at java.lang.Thread.run(Thread.java:750)
Caused by: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 132200 column 23 path $.
	at com.google.gson.stream.JsonReader.beginObject(JsonReader.java:385)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:215)
	... 10 more
Parsing data...
com.kitsunecode.mms.core.entities.exceptions.StartFailedException: Critical error while creating the adapter: NullPointerException: null
	at com.kitsunecode.mms.core.utils.Util.getWaifuFromAdapterName(Util.java:249)
	at com.kitsunecode.mms.core.Main.lambda$initialize$0(Main.java:37)
	at com.kitsunecode.mms.core.utils.Util.catchMoeMoeExceptionsAndExit(Util.java:339)
	at com.kitsunecode.mms.core.Main.initialize(Main.java:35)
	at com.kitsunecode.mms.core.Main.main(Main.java:45)
Caused by: com.kitsunecode.mms.core.entities.exceptions.StartFailedException: NullPointerException: null
	at com.kitsunecode.mms.core.adapters.IWaifuAdapter.init(IWaifuAdapter.java:68)
	at com.kitsunecode.mms.core.utils.Util.getWaifuFromAdapterName(Util.java:234)
	... 4 more
Caused by: java.lang.NullPointerException
	at com.kitsunecode.mms.core.adapters.impl.Arknights.loadFromCustomSource(Arknights.java:85)
	at com.kitsunecode.mms.core.adapters.IWaifuAdapter.init(IWaifuAdapter.java:50)
	... 5 more
```

--- a/pom.xml
+++ b/pom.xml
@@ -6,7 +6,7 @@
 
     <groupId>com.kitsunecode</groupId>
     <artifactId>moe-moe-secretary</artifactId>
-    <version>1.2.1</version>
+    <version>1.2.2</version>
     <packaging>jar</packaging>
 
     <properties>

--- a/src/main/java/com/kitsunecode/mms/core/adapters/impl/adapterentities/arknights/charword/CharWords.java
+++ b/src/main/java/com/kitsunecode/mms/core/adapters/impl/adapterentities/arknights/charword/CharWords.java
@@ -1,0 +1,6 @@
+package com.kitsunecode.mms.core.adapters.impl.adapterentities.arknights.charword;
+
+import java.util.HashMap;
+
+public class CharWords extends HashMap<String, Charword> {
+}

--- a/src/main/java/com/kitsunecode/mms/core/adapters/impl/adapterentities/arknights/charword/Charword.java
+++ b/src/main/java/com/kitsunecode/mms/core/adapters/impl/adapterentities/arknights/charword/Charword.java
@@ -4,7 +4,7 @@ import com.kitsunecode.mms.core.entities.Dialog;
 
 public class Charword {
 
-    private static final String VOICE_URL = "https://raw.githubusercontent.com/Aceship/AN-EN-Tags/master/etc/voice/%s.mp3";
+    private static final String VOICE_URL = "https://raw.githubusercontent.com/Aceship/Arknight-voices/main/voice/%s.mp3";
 
     private String charWordId;
     private String charId;

--- a/src/main/java/com/kitsunecode/mms/core/adapters/impl/adapterentities/arknights/charword/CharwordMap.java
+++ b/src/main/java/com/kitsunecode/mms/core/adapters/impl/adapterentities/arknights/charword/CharwordMap.java
@@ -1,22 +1,23 @@
 package com.kitsunecode.mms.core.adapters.impl.adapterentities.arknights.charword;
 
-import com.kitsunecode.mms.core.adapters.impl.adapterentities.arknights.character.Character;
-import com.kitsunecode.mms.core.utils.Util;
-
-import java.util.HashMap;
 import java.util.List;
 import java.util.stream.Collectors;
 
-public class CharwordMap extends HashMap<String, Charword> {
+import com.kitsunecode.mms.core.adapters.impl.adapterentities.arknights.character.Character;
+import com.kitsunecode.mms.core.utils.Util;
+
+public class CharwordMap {
+
+    private CharWords charWords;
 
     public static CharwordMap fromJson(String json) {
         return Util.getGSON().fromJson(json, CharwordMap.class);
     }
 
     public List<Charword> ofCharacter(Character character) {
-        return entrySet().parallelStream()
+        return charWords.entrySet().parallelStream()
                 .filter(e -> e.getValue().getCharId().equalsIgnoreCase(character.getId()))
-                .map(Entry::getValue)
+                .map(e -> e.getValue())
                 .collect(Collectors.toList());
     }
 

--- a/src/main/java/com/kitsunecode/mms/core/adapters/impl/adapterentities/arknights/skin/Skin.java
+++ b/src/main/java/com/kitsunecode/mms/core/adapters/impl/adapterentities/arknights/skin/Skin.java
@@ -2,7 +2,7 @@ package com.kitsunecode.mms.core.adapters.impl.adapterentities.arknights.skin;
 
 public class Skin {
 
-    private static final String SKIN_URL = "https://raw.githubusercontent.com/Aceship/AN-EN-Tags/master/img/characters/%s.png";
+    private static final String SKIN_URL = "https://raw.githubusercontent.com/Aceship/Arknight-Images/main/characters/%s.png";
 
     private String skinId;
     private String charId;

